src/EnergyPlus/ThermalComfort.cc

```diff
@@ -2888,14 +2888,21 @@ namespace ThermalComfort {
             const bool epwFileExists = FileSystem::fileExists(ioFiles.inputWeatherFileName.fileName);
             readStat = 0;
             if (epwFileExists) {
+                // determine number of days in year
+                int DaysInYear;
+                if (DataEnvironment::CurrentYearIsLeapYear) {
+                    DaysInYear = 366;
+                } else {
+                    DaysInYear = 365;
+                }
                 auto epwFile = ioFiles.inputWeatherFileName.open("CalcThermalComfortAdaptiveCEN15251");
-                for (i = 1; i <= 9; ++i) { // Headers
+                for (i = 1; i <= 8; ++i) { // Headers
                     epwFile.readLine();
                 }
                 jStartDay = DayOfYear - 1;
                 calcStartDay = jStartDay - 7;
                 if (calcStartDay > 0) {
-                    calcStartHr = 24 * (calcStartDay - 1) + 1;
+                    calcStartHr = 24 * calcStartDay + 1;
```

Comment:
So I guess the `- 1` wasn't needed because we already offset DayOfYear by 1 a couple lines up?

Reply:
I am not sure about the original intent, but I think this could be one explanation.

Reply:
`24 * (calcStartDay - 1)` is a conversion to hours. Why exactly was this changed when calcStartDay wasn't changed?

Reply:
Because it was wrong by offsetting one excessive day here. This was also an issue with the ASHRAE comfort calculation function above, which was fixed in #6474.

Reply:
I see, this code is wrapping the epw to 24*calcStartDay as the number of hours last year and +1 is the first hour of this year. If you pulled develop this would not show as a diff. Never mind that right now.